MinecraftClient/Commands/Entitycmd.cs

```diff
@@ -160,12 +188,40 @@ public override string Run(McClient handler, string command, Dictionary<string,
                         {
                             if (entity2.Value.Type == interacttype)
                             {
-                                GetEntityInfoShort(response, entity2.Value);
+                                response.AppendLine(GetEntityInfoShort(entity2.Value));
                             }
                         }
                         response.Append(GetCmdDescTranslated());
                         return response.ToString();
                     }
+                    else if (args[0] == "near")
+                    {
+                        if (args.Length > 1)
+                        {
+                            if (Enum.TryParse(args[1], true, out EntityType entityType))
+                            {
+                                if (TryGetClosetEntity(handler.GetEntities(), handler.GetCurrentLocation(), entityType, out Entity? closest))
+                                    return InteractionWithEntity(handler, closest, args.Length > 2 ? args[2].ToLower() : "list");
+                                else
+                                    return Translations.cmd_entityCmd_not_found;
+                            }
+                            else if (int.TryParse(args[0], NumberStyles.Any, CultureInfo.CurrentCulture, out int entityID2))
```

Comment:
I think there is a typo here it should be
```csharp
int.TryParse(args[1], NumberStyles.Any, CultureInfo.CurrentCulture, out int entityID2)
```
otherwise it will always try to parse "near"

Reply:
> I think there is a typo here it should be
> 
> ```cs
> int.TryParse(args[1], NumberStyles.Any, CultureInfo.CurrentCulture, out int entityID2)
> ```
> 
> otherwise it will always try to parse "near"

Yes, it's fixed in the latest version.